source/frontend/src/features/properties/map/project/add/AddProjectForm.tsx

```diff
@@ -19,66 +22,82 @@ export interface IAddProjectFormProps {
-                  <Select
-                    field="projectStatusType"
-                    options={projectStatusOptions}
-                    placeholder="Select..."
-                  />
-                </SectionField>
-                <SectionField label="MoTI region" required={true}>
-                  <Select field="region" options={projectRegionOptions} placeholder="Select..." />
-                </SectionField>
-                <SectionField label="Project summary">
-                  <MediumTextArea field="summary" />
-                </SectionField>
-              </Section>
-            </Container>
-          </>
-        )}
-      </Formik>
-    );
-  },
-);
+  return (
+    <Formik<ProjectForm>
+      enableReinitialize
+      innerRef={props.formikRef}
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+      onSubmit={handleSubmit}
+    >
+      {formikProps => (
+        <StyledFormWrapper>
+          <Form>
+            <Section>
+              <StyledRow className="no-gutters py-4 mb-5">
+                <Col>
+                  <p>
+                    Before creating a project, <Link to={'/project/list'}>do a search</Link> to
+                    ensure the the project you're creating doesn't already exist.
+                  </p>
+                </Col>
+              </StyledRow>
+              <SectionField label="Project name" required labelWidth="2">
+                <Input field="projectName" />
+              </SectionField>
+              <SectionField label="Project number" labelWidth="2">
+                <Input field="projectNumber" placeholder="if known" />
+              </SectionField>
+              <SectionField label="Status" required labelWidth="2">
+                <Select
+                  field="projectStatusType"
+                  options={projectStatusOptions}
+                  placeholder="Select..."
+                />
+              </SectionField>
+              <SectionField label="MoTI region" required labelWidth="2">
+                <Select field="region" options={projectRegionOptions} placeholder="Select..." />
+              </SectionField>
+              <SectionField label="Project summary" labelWidth="12">
+                <MediumTextArea field="summary" />
+              </SectionField>
+            </Section>
+            <ProductsArrayForm formikProps={formikProps} field="products" />
+          </Form>
+        </StyledFormWrapper>
+      )}
+    </Formik>
+  );
+};
 
 export default AddProjectForm;
 
+const StyledFormWrapper = styled.div`
+  background-color: ${props => props.theme.css.filterBackgroundColor};
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+`;
+
+const StyledRow = styled(Row)`
+  border-bottom-style: solid;
+  border-bottom-color: grey;
+  border-bottom-width: 0.1rem;
+  text-align: left;
+`;
+
 export const MediumTextArea = styled(TextArea)`
-  textarea.form-control {
+  /*textarea.form-control {
```

Comment:
can we remove this? also surprised this compiles lol, aren't those comments within the ``?